geth/node/manager.go

```diff
@@ -54,10 +54,6 @@ func NewNodeManager() *NodeManager {
 
 // StartNode start Status node, fails if node is already started
 func (m *NodeManager) StartNode(config *params.NodeConfig) (<-chan struct{}, error) {
-	if m == nil {
```

Comment:
Hm, why these changes appeared in this PR?

Reply:
Since @tiabc mention our need to stop checking such logic which should rather be validated in tests, felt it best to remove this, is there a problem? Do we wish to move it into a PR of it's own?

Reply:
Up to @tiabc, but I saw a few of his comments when he insisted on not introducing refactoring changes into feature PRs.

Reply:
Yeah, if you just noticed something in unrelated part of code, just let it go and add to a refactoring note. The goal here is to ease merging.
In this case they're unrelated and should have been skipped but we're not going to discuss it and waste time, let's just merge it as soon as possible and keep it in mind next time.

Reply:
Alright!